File: FizzBuzz/FizzBuzz.java
//Fizz Buzz
//For multiples of 3 --> Fizz 
//For multiples of 5 --> Buzz 
//For multiples of both --> FizzBuzz

public static void main(String [] args) 
{
	//count and print from 1 to 100
	for(int i = 0; i <= 100; i++)
	{
		//check multiples 
		if(i % 15 == 0)
			System.out.println("FizzBuzz");

		else if(i % 3 == 0)
			System.out.println("Fizz");

		else  if (i % 5 == 0)
			System.out.println("Buzz");

		else 
			Sysem.out.println(i);
	}
}

File: FizzBuzz/FizzBuzz.py
#Fizz Buzz
#For multiples of 3 --> Fizz 
#For multiples of 5 --> Buzz 
#For multiples of both --> FizzBuzz

def main(): 
	#count and print numbers 1 to 100

	for i in range (1, 100): 
		if i % 15 == 0: 
			print ("FizzBuzz")

		elif i % 3 == 0: 
			print("Fizz")

		elif i % 5 == 0: 
			print("Buzz")

		else: 
			print(i)
main()
